<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bamboo Map Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>165</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>165</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>Paint</string>
            </property>
            <widget class="QComboBox" name="comboTileset">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>121</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QComboBox" name="comboTileID">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>121</width>
               <height>22</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>A</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>B</string>
              </property>
             </item>
            </widget>
            <widget class="QComboBox" name="comboBrush">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>80</y>
               <width>121</width>
               <height>22</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>Rectangle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Eclipse</string>
              </property>
             </item>
            </widget>
            <widget class="QSpinBox" name="spinRadius">
             <property name="geometry">
              <rect>
               <x>71</x>
               <y>110</y>
               <width>61</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>15</x>
               <y>114</y>
               <width>46</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Radius</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="title">
             <string>Resize</string>
            </property>
            <widget class="QSpinBox" name="spinWidth">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>20</y>
               <width>61</width>
               <height>22</height>
              </rect>
             </property>
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>300</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinHeight">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>50</y>
               <width>61</width>
               <height>22</height>
              </rect>
             </property>
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>300</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>46</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Width</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>46</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btnResize">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>80</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MapView" name="graphicsView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionOption"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCreate"/>
   <addaction name="actionEdit"/>
   <addaction name="actionTerrain"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Folder.png</normaloff>:/icon/resources/childish_Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open(CTRL+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_File.png</normaloff>:/icon/resources/childish_File.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New(CTRL+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Floppy-Disk.png</normaloff>:/icon/resources/childish_Floppy-Disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save(CTRL+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Gears.png</normaloff>:/icon/resources/childish_Gears.png</iconset>
   </property>
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/cut.png</normaloff>:/icon/resources/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut(CTRL+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_News.png</normaloff>:/icon/resources/childish_News.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy(CTRL+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Clipboard.png</normaloff>:/icon/resources/childish_Clipboard.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste(CTRL+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Cross.png</normaloff>:/icon/resources/childish_Cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete (DEL)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Tool.png</normaloff>:/icon/resources/childish_Tool.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit object(E)</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionCreate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Image.png</normaloff>:/icon/resources/childish_Image.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="toolTip">
    <string>Create object(R)</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionTerrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resources/childish_Brush.png</normaloff>:/icon/resources/childish_Brush.png</iconset>
   </property>
   <property name="text">
    <string>terrain</string>
   </property>
   <property name="toolTip">
    <string>Paint terrain(T)</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>mapview.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
